scripts/tokenomics-emulation/emulations/validators.emulation.ts

```diff
@@ -0,0 +1,37 @@
+import { IEmulation } from "./emulation.interface";
+import Network from "../network";
+import Accounts from "../accounts";
+import Validator from "../validator";
+import fs from "fs";
+
+class ValidatorsEmulation implements IEmulation {
+  constructor(
+    private readonly config,
+    private readonly account: Accounts
+  ) {}
+
+  public async run(): Promise<void> {
+    const genericValidator = this.config.network.validators.amount;
+    const commissionValue = this.config.network.nominators.commission;
+    const stashBond = this.config.network.nominators.stash_stake;
```

Comment:
It should be `stash_stake` minus the existential deposit.

Reply:
added calculation to substrate exsistential deposit